### DISCUSSION TITLE
This is nuts!

### DISCUSSION THREAD

**Discussion by liquidcarbon** [liquidcarbon/liquidcarbon#2] [Polls]
Look, a poll, with more than 4 options, running indefinitely!

<img src="https://github.com/liquidcarbon/thoughts/assets/47034358/9577aafd-67c3-44fb-acfd-f2a0f310b177" width="400">

**Comment by lemasterdj** [comment]
"Markdown is supported."

Yes, there you go—all the customization power!

Wishing you best, Alexander.
